### ISSUE TITLE
Berlin Page Tests

### ISSUE THREAD

**Issue by bensternthal** [mozilla/bedrock#5637]
This work request is to test the Berlin page phase 2 in #5706 w. video (vs.) a new variation on the Berlin page phase 2 w. video in different layout (top of page) that will need to be created to enable this test

[Test write up from manapage template](https://mana.mozilla.org/wiki/pages/viewpage.action?pageId=77893531)

* [Data Studio Report](https://datastudio.google.com/u/0/org/Ra9O2QIYT8We-1pOmnS_6w/reporting/0B-vqNw0Uhi8RR3ZGdHBTZF9TWUU/page/uGB)

**Comment by ejregithub** [comment]
This blocked pending reply from Guy Fitzpatrick as to the details of the test itself.

**Comment by bensternthal** [comment]
@ejregithub when you are back from pto.. is this still blocked?  If so do we have a date when Guy will provide whatever details are missing?

**Comment by ejregithub** [comment]
This is unblocked. We can run the test when the variation is created placing the video at the top of the page. @dzingeek  needs to look at the proposed to design to determine if the video should occupy more space in the page or be seen in a modal (or other).

**Comment by dzingeek** [comment]
We should treat the video the same way in version B as we do in version A (no model, play where it is). As long as the user has control to make the video full screen if they choose.

**Comment by alexgibson** [comment]
@ejregithub do we have any design assets for the background featured in variation b? I think this is the only thing we need to start work on this.

**Comment by ejregithub** [comment]
@alexgibson - please see assets [here](https://drive.google.com/file/d/1rJXD56N538wa1KLtHaomGI6pEWaqTms_/view)

**Comment by alexgibson** [comment]
This is now up on demo: https://bedrock-demo-agibson.oregon-b.moz.works/de/firefox/new/?xv=berlin&v=b

**Comment by guyfitzpatrick** [comment]
@alexgibson looks good, thanks!

We just need to swap out the thumbnail on the video on both pages and we are then good to go: https://drive.google.com/open?id=1NMuPIkcNRiAjfZVGGQZ2jcphEBXUCaHz

**Comment by alexgibson** [comment]
> We just need to swap out the thumbnail on the video on both pages and we are then good to go: https://drive.google.com/open?id=1NMuPIkcNRiAjfZVGGQZ2jcphEBXUCaHz

The placeholder image is now updated. Code review has also been finished, so the last thing we're waiting on here is GA reporting.

**Comment by guyfitzpatrick** [comment]
Thank you, @alexgibson!

**Comment by peterGerman** [comment]
@alexgibson Looks good to me.

**Comment by alexgibson** [comment]
> @alexgibson I still see the old thumbnail on all Berlin landing pages and version b (video in header) has now just become a duplicate of version a (video under header). Please could you double check?

The page is not yet live (it only just merged). It will go live in our next deployment later today.

**Comment by guyfitzpatrick** [comment]
@alexgibson great, thanks!

**Comment by alexgibson** [comment]
These pages are now line:

https://www.mozilla.org/de/firefox/new/?xv=berlin&v=a
https://www.mozilla.org/de/firefox/new/?xv=berlin&v=b

I'll start the A/B test shortly and update here when it's running.

**Comment by guyfitzpatrick** [comment]
@alexgibson thanks, look forward to the results

**Comment by alexgibson** [comment]
> @alexgibson thanks, look forward to the results

Experiment is now live! /cc @peterGerman
